.github/workflows/main.yml

```diff
@@ -11,6 +11,22 @@ on:
   pull_request:
 
 jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v3
+        with:
+          any-of-labels: "Status: Waiting for Customer"
+          days-before-issue-stale: 14
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 14 days with no activity."
+          close-issue-message: "This issue was closed because it has been inactive for 14 days since being marked as stale."
```

Comment:
Nit: I think we should provide a bit more info in the message.

Reply:
So here are my proposals:

*stale-issue-message*
> This issue has been marked as "stale" due to its inactivity for the last 14 days.
Please, add a new comment to keep the conversation ongoing; otherwise, the issue will be closed in two weeks.

*close-issue-message*
> This issue has been closed due to its inactivity for the last month.
Please, feel free to reopen it and add a new comment in case you think the problem has not been solved.

Reply:
@CarlosCortizasCT thanks for your suggestion. Updated it just a little bit, added to the new commit.